homeassistant/components/fritz/common.py

```diff
@@ -204,19 +208,25 @@ def __init__(self, mac, name=None):
         self._last_activity = None
         self._connected = False
 
-    def update(self, dev_info, dev_home):
+    def update(self, dev_info, dev_home, consider_home):
         """Update device info."""
-        utc_point_in_time = dt_util.utcnow()
+        utc_point_in_time = dt_util.now()
```

Comment:
`dt_util.now` isn't always utc. Its in local time if they changed the time zone

Reply:
as this is used in a attribute we should display the local time:

`Last time reachable:   2021-05-22T22:57:20+02:00`

Reply:
The UI should translate the time to local time if its in isoformat

Reply:
Unfortunately, it doesn't if it's an attribute. This is why I changed it

Reply:
@MartinHjelmare can you give your advice here please?

Reply:
Looks like we only convert it to local time if its DEVICE_CLASS_TIMESTAMP <https://github.com/home-assistant/frontend/blob/dev/src/panels/lovelace/entity-rows/hui-sensor-entity-row.ts#L73>

We should leave the backend as is and make a PR to localize isotime attributes instead as localizing is a job for the frontend/display layer in https://github.com/home-assistant/frontend/blob/2ee92f48e6eaa0de017822e99d6fa19d1ca12d27/src/components/ha-attributes.ts#L108

Reply:
Times in the state machine should *always* be UTC.

Reply:
Reverted, should be good to go now.

Side note: we need to check other components for wrong usage of now()